.circleci/config.yml

```diff
@@ -537,6 +537,73 @@ jobs:
     working_directory: ~/kubernetes-monitor
+  push_operator_to_community_operators:
+    description: |
+      Packages a new Operator and pushes it to Snyk's fork of
+      the OpenShift community-operators.
+    executor: redhat-openshift/default
+    steps:
+    - checkout
+    - add_ssh_keys:
+        fingerprints:
+        - 06:c3:d4:10:0d:ef:37:6c:ec:b9:fb:6e:ed:09:e7:40
+    - run:
+        command: |
+          python -m pip install requests pyyaml
+        name: Install required packages
+    - install_python_requests
+    - run:
+        command: |
+          set -xeo pipefail
+          LAST_OPERATOR_VERSION=$(python ./scripts/operator/get-last-published-operator-version.py)
+          echo "export LAST_OPERATOR_VERSION=${LAST_OPERATOR_VERSION}" >> $BASH_ENV
+        name: Get last released Operator version
+    - run:
+        command: |
+          LATEST_TAG_WITH_V=`git describe --abbrev=0 --tags ${CIRCLE_SHA1}`
+          LATEST_TAG=${LATEST_TAG_WITH_V:1}
+          NEW_OPERATOR_VERSION=${LATEST_TAG}
+          echo "export NEW_OPERATOR_VERSION=${NEW_OPERATOR_VERSION}" >> $BASH_ENV
+        name: Get new Operator version
+    - run:
+        command: |
+          ./scripts/operator/package-operator.sh "${NEW_OPERATOR_VERSION}" "${NEW_OPERATOR_VERSION}" "${NEW_OPERATOR_VERSION}" "${LAST_OPERATOR_VERSION}"
+        name: Package Operator
+    - run:
+        command: |
+          set -xeo pipefail
+
+          CURRENT_DIRECTORY=$(pwd)
+          COMMUNITY_OPERATORS_UPSTREAM_LOCATION="${CURRENT_DIRECTORY}/community-operators"
+          DEPLOY_LOCATION="${COMMUNITY_OPERATORS_UPSTREAM_LOCATION}/community-operators"
+          OPERATOR_LOCATION="${CURRENT_DIRECTORY}/snyk-operator/deploy/olm-catalog/snyk-operator"
+
+          # Configure git user and gpg key
+          echo "${OPENSHIFT_OPERATOR_SIGNING_KEY_BASE64}" | base64 -d | gpg --import
+          git config --global commit.gpgsign true
+          git config --global user.signingkey "${OPENSHIFT_OPERATOR_SIGNING_KEY_ID}"
+          git config --global user.email "${OPENSHIFT_OPERATOR_GITHUB_EMAIL}"
+          git config --global user.name "${OPENSHIFT_OPERATOR_GITHUB_NAME}"
+
+          # Clone Community Operators repo from Snyk
+          git clone https://github.com/snyk/community-operators.git $COMMUNITY_OPERATORS_UPSTREAM_LOCATION
+          cd "${COMMUNITY_OPERATORS_UPSTREAM_LOCATION}"
+          git checkout -b snyk/snyk-operator-v${NEW_OPERATOR_VERSION}
+
+          # Copy new release to branch
+          cp -r "${OPERATOR_LOCATION}/${NEW_OPERATOR_VERSION}" "${DEPLOY_LOCATION}/snyk-operator/."
+          cp "${OPERATOR_LOCATION}/snyk-operator.package.yaml" "${DEPLOY_LOCATION}/snyk-operator/."
+
+          # Create the signed commit and push
+          git add "${DEPLOY_LOCATION}/snyk-operator/*"
+          git commit -s -m "Upgrade snyk-operator to version ${NEW_OPERATOR_VERSION}"
+          git push --set-upstream origin --force snyk/snyk-operator-v${NEW_OPERATOR_VERSION}
```

Comment:
I think I missed something here. But we're only pushing a new branch to the community operator repo, right? But we're not opening a Pull Request. This step is still a manual process, right?

Reply:
Correct, I'll update the job description.
I'm not yet sure what's the effort to open a PR to the community-operators, may have to do this change as a separate PR